pkg/agent/util/net.go

```diff
@@ -38,6 +38,8 @@ const (
 
 	FamilyIPv4 uint8 = 4
 	FamilyIPv6 uint8 = 6
+
+	BridgedUplinkPrefix = "b-"
```

Comment:
@wenyingd Which prefix you used in BM/VM?

Reply:
Prefix `phy` is used in VM support case. But I am thinking maybe `ant` is better to represent it is renamed by Antrea, just like antctl? What's your thought @gran-vmv @mengdie-song ?

Reply:
@tnqn

Reply:
Could we use `antrea-` (as antrea-gw0, antrea-tun0), or you would choose a shorter prefix due to the interface length limit?

Reply:
I know NSX-T appends a single letter to the interface name (I remember it is `~`, like `eth0~`).

Reply:
Linux has a property `IFNAMSIZ` to limit the network interface name length to 15 bytes, thus I prefer to use short prefix/suffix, and I feel that suffix `~` is better. @wenyingd @tnqn @mengdie-song

Reply:
I am OK with the suffix.

Reply:
In BM/VM and this PR, we'll restrict the uplink interface name length, thus we'd better choose a shorter prefix/suffix to improve the compatibility.